install/cluster-kube-apiserver-operator/install.yaml

```diff
@@ -0,0 +1,111 @@
+apiVersion: template.openshift.io/v1
```

Comment:
why a template? We should only use kube core types.

Reply:
agree, just do list and hardcode the namespace

i think we can still process this template via client, so we don't need openshift api server

Reply:
This is for cluster up.  Cluster up can easily take templates.  It can't easily take lists.